specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdates.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "updateId": "35428702-5784-4ba4-a6e0-5222258b5411",
+    "api-version": "2018-06-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/mySubscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines/myVMName/providers/Microsoft.GuestConfiguration/softwareUpdates/35428702-5784-4ba4-a6e0-5222258b5411",
+        "name": "35428702-5784-4ba4-a6e0-5222258b5411",
+        "type": "Microsoft.GuestConfiguration/softwareUpdates",
+        "properties": {
+    "updateState": "Needed",
+    "availableOnComputerSince": "2018-11-07T00:02:21Z",
+    "approved": "true",
+    "packageRepository": "",
+    "supercededUpdateIds": "",
+    "isSupercededUpdate": "",
+    "deploymentAction": "",
+    "kbid": "2267602",
+    "kbRevisionNumber": "200",
+    "publishedDate": "2018-11-07T00:00:00Z",
+    "productName": "Windows Defender",
+    "productVersion": "",
+    "productArch": "",
+    "title": "Definition Update for Windows Defender Antivirus - KB2267602 (Definition 1.279.1373.0)",
+    "classification": "Definition Updates",
+    "packageClassification": "",
+    "packageSeverity": "",
+    "msrcSeverity": "",
+    "externalURL": "https://support.microsoft.com/en-us/help/2267602",
+    "msrcBulletinId": "",
+    "cveNumbers": "",
+    "rebootBehavior": "NeverReboots",
+    "optional": "false",
+    "autoSelectOnWebsites": "",
+    "browseOnly": "",
+    "lastRefreshDate": "2018-11-11T00:00:00Z"
```

Comment:
This property is no longer present. See https://msazure.visualstudio.com/One/_git/mgmt-automation-agentdataservice?path=%2Fsrc%2FSamples%2FPayload%2FSoftwareUpdatesPayload.json&version=GBmaster for the most up-to-date list

Reply:
GuestConfig RP is the source of truth for ARM response here. https://msazure.visualstudio.com/One/_git/Mgmt-Automation-GuestConfiguration?path=%2Fsrc%2FGuestConfiguration.ViewModel%2FSoftwareUpdateV1.cs&version=GBmaster

Reply:
The input and output objects appear to have diverged. I created this task to for someone to go back and double-check that they match: https://msazure.visualstudio.com/One/_workitems/edit/3696270

Reply:
resolved with sync